rpcs3/rpcs3qt/emu_settings.cpp

```diff
@@ -632,3 +633,163 @@ void emu_settings::OpenCorrectionDialog(QWidget* parent)
 		cfg_log.success("You need to save the settings in order to make these changes permanent!");
 	}
 }
+
+QString emu_settings::GetLocalizedSetting(const QString& original, SettingsType type, int index)
+{
+	switch (type)
+	{
+	case emu_settings::SPUBlockSize:
+		switch (static_cast<spu_block_size_type>(index))
+		{
+		case spu_block_size_type::safe: return tr("Safe");
+		case spu_block_size_type::mega: return tr("Mega");
+		case spu_block_size_type::giga: return tr("Giga");
+		}
+		break;
+	case emu_settings::EnableTSX:
+		switch (static_cast<tsx_usage>(index))
+		{
+		case tsx_usage::disabled: return tr("Disabled");
+		case tsx_usage::enabled: return tr("Enabled");
+		case tsx_usage::forced: return tr("Forced");
+		}
+		break;
+	case emu_settings::Renderer:
+		switch (static_cast<video_renderer>(index))
+		{
+		case video_renderer::null: return tr("Null");
```

Comment:
There is already logic somewhere that converts "Null" to "Disable video output", can't it be merged into this function? Same goes for the other `tr("Null")`'s in this switch.

Reply:
I removed the useless string replacements.

Reply:
Any chance of naming the keyboard, microphone etc. null handlers to Disconnected (or something more fitting)? It's bit out of scope for this PR but it'd be more consistent and would save translations from trouble of translating the word Null.